### ISSUE TITLE
Calibrating tapster on Android

### ISSUE THREAD

**Issue by bebef1987** [appium-boneyard/robots#6]
Is there an app to calibrate Tapster on Android ?

I'm trying to use tapster on different platforms but need to calibrate it. Any suggestions on how to do it?

**Comment by penguinho** [comment]
Yes, https://github.com/appium/robots/tree/master/apps/ios/src/RobotCalibration

Newer code for this is in my fork of the tapster repo.

https://github.com/penguinho/tapsterbot

**Comment by bebef1987** [comment]
@penguinho That still points to an IOS project. As I don't have access to a Mac and XCode to build and run that app on IOS I'm unable to use the suggested code.

**Comment by penguinho** [comment]
You'll have to build an equivalent android app, I also believe there used to be a web app as well. You'll have to ask @hugs

**Comment by bebef1987** [comment]
I'm building an Firefox OS app and porting the client to Python as I'm an little rusty with JS
